firestore-service.js

```diff
@@ -50,8 +52,10 @@ function initializeFirestore(keys) {
 
 async function getData({ pathname, query }) {
   try {
-    const { id, limit } = query;
-    let data = firestore.collection(pathname);
+    const id = getIdFromPath(pathname);
+    const path = id ? getCollectionPath(pathname) : pathname;
```

Comment:
what if we have more than one collection in `pathname` something like: `companies/2/users/7` ?

Reply:
It won't be a problem, firebase `.collection()` allows for you to give a path as complex as it may be with the condition that the last element it will return would be a collection. So for example you can do this
`companies/2/employees` And it will return a collection with all the employees. of the company n2
But you can't do this
`companies/2/employees/4` Because it will return a single doc. For this to work you will have to make a query like this 
```
db.collection(`companies/2/employees`).doc(4)
```

Reply:
greatttt! understood! 👍